netbox_onboarding/netbox_keeper.py

```diff
@@ -30,6 +31,24 @@
 PLUGIN_SETTINGS = settings.PLUGINS_CONFIG["netbox_onboarding"]
 
 
+def object_match(obj, search_array):
+    """Used to search models for multiple criteria."""
+    try:
+        result = obj.objects.get(**search_array[0])
+        return result
+    except obj.DoesNotExist as error:
+        if PLUGIN_SETTINGS['object_match_strategy'] == "loose":
+            for search_array_element in search_array[1:]:
+                try:
+                    result = obj.objects.get(**search_array_element)
+                    return result
+                except obj.DoesNotExist:
+                    pass
+                except obj.Multiple:
+                    raise OnboardException(reason="fail-general", message=f"ERROR multiple objects found")
```

Comment:
In the OnboardException's error message try to print the name of the object too. (`obj`)

Reply:
Probably also include `**search_array_element` so we know which one of the searches was the culprit.